src/components/TextField/TextField.tsx

```diff
@@ -1,13 +1,16 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import '../../yamui';
 import * as React from 'react';
+import { join } from '../../util/classNames';
 import { getBaseTextFieldProps, BaseTextFieldProps } from '../BaseTextField';
 import DebouncedOnChange, { DebouncedOnChangeProps, DebouncedOnChangePrivateProps } from '../../util/DebouncedOnChange';
 import { TextField as FabricTextField, ITextField } from 'office-ui-fabric-react/lib/TextField';
 
 import '../BaseTextField/BaseTextField.css';
 import './TextField.css';
 
+export { ITextField };
```

Comment:
Why is this necessary?

Reply:
it's used by EditableText when storing the underlying TextField ref (in the next PR). but actually i can probably use our TextField type directly since @chrismohr implemented the debounced wrapper to expose it. i'll update this 👍

Reply:
removed. in the next PR #313 i added an interface for TextField to implement which declares our `focus` method.